cmd/go-filecoin/client.go

```diff
@@ -107,48 +114,101 @@ be 2, 1 hour would be 120, and 1 day would be 2880.
 	Arguments: []cmdkit.Argument{
 		cmdkit.StringArg("miner", true, false, "Address of miner to send storage proposal"),
 		cmdkit.StringArg("data", true, false, "CID of the data to be stored"),
-		cmdkit.StringArg("ask", true, false, "ID of ask for which to propose a deal"),
-		cmdkit.StringArg("duration", true, false, "Time in blocks (about 30 seconds per block) to store data"),
+		cmdkit.StringArg("start", true, false, "Chain epoch at which deal should start"),
+		cmdkit.StringArg("end", true, false, "Chain epoch at which deal should end"),
+		cmdkit.StringArg("price", true, false, "Price of deal in FIL (e.g. 0.01)"),
```

Comment:
It would be good to clarify whether this is total, per-byte-epoch, or what other units.